distribution/src/database/openfire_db2.sql

```diff
@@ -234,6 +235,11 @@ CREATE TABLE ofMucRoomProp (
   CONSTRAINT ofMucRoomProp_pk PRIMARY KEY (roomID, name)
 );
 
+CREATE TABLE ofMucRoomRetiree (
+  serviceID           INTEGER       NOT NULL,
+  name                VARCHAR(50)   NOT NULL,
```

Comment:
It's almost a shame to not add a bit of metadata here, like the timestamp of the retirement and the JID of the entity that did the retiring, the 'destroy' message and alternative JID if those were used. All of this would be optional (apart from the timestamp, which can just be a default).

Reply:
I have added alternateJID, reason, and retired-at timestamp.